model/parVarSheet.cc

```diff
@@ -129,106 +131,115 @@ namespace minsky
                     cairo_clip(cairo);             
                    
                     y0=h+3.1*rowHeight;                 
-			      }
-			    else
-			      {
+                  }
+                else
+                  { 
                     cairo_move_to(cairo,x,y-1.5*rowHeight);
                     pango.setMarkup(latexToPango(value->name)+":");
```

Comment:
The remainder of the code in this function is too difficult to review, but IIUC it is just reformatted from what was before.

Reply:
Yes. I've seen you've made improvements in the Sheet class and I've been comparing that code with mine to make improvements here too.